dali/python/backend_impl.cc

```diff
@@ -613,6 +613,17 @@ PYBIND11_MODULE(backend_impl, m) {
         [](Pipeline *p) {
           p->Build();
           })
+    .def("SetExecutionTypes",
+        [](Pipeline *p, bool exec_pipelined, bool exec_separated, bool exec_async) {
+          p->SetExecutionTypes(exec_pipelined, exec_separated, exec_async);
+        },
+        "exec_pipelined"_a = true,
```

Comment:
I don't understand those three lines

Reply:
Default values?

Reply:
I followed the bindings to constructor, those are user defined literals from Pybind, used as a shortcut to create keyword args. Maybe it's a bit of an overkill to use kwargs here.